bookstore/src/main/webapp/WEB-INF/tiles.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" ?>  
+<!DOCTYPE tiles-definitions PUBLIC  
+      "-//Apache Software Foundation//DTD Tiles Configuration 3.0//EN"  
+      "http://tiles.apache.org/dtds/tiles-config_3_0.dtd">
+<tiles-definitions>
+
+	<!-- Base Define -->
+	<definition name="bookstore.definition" template="/WEB-INF/layouts/classic.jsp">
+		<put-attribute name="title" value="" />
+		<put-attribute name="header" value="/WEB-INF/template/header.jsp" />
+		<put-attribute name="navigation" value="/WEB-INF/template/navigation.jsp" />
+		<put-attribute name="body" value="" />
+		<put-attribute name="footer" value="/WEB-INF/template/footer.jsp" />
+	</definition>
+
+	<!-- Home Page -->
+	<definition name="homePage" extends="bookstore.definition">
+		<put-attribute name="title" value="Ngôi Nhà Sách" />
+		<put-attribute name="body" value="/WEB-INF/view/home.jsp" />
+	</definition>
+
+	<!-- ContactUs Page -->
+	<definition name="contactusPage" extends="bookstore.definition">
+		<put-attribute name="title" value="Liên Hệ" />
+		<put-attribute name="body" value="/WEB-INF/view/contactus.jsp" />
+	</definition>
+	
+	<!-- Login Page -->
+	<definition name="loginPage" extends="bookstore.definition">
+		<put-attribute name="title" value="login" />
```

Comment:
The title should be changed from "login" to "User Login"
@ronaldolima99

Reply:
done